tests/test_coadd.py

```diff
@@ -128,11 +127,12 @@ def testNoiseCoadd(self):
     def testFilters(self):
         """Test that the coadd filter is set correctly
         """
-        filterPolicyFile = pexPolicy.DefaultPolicyFile("afw", "SdssFilters.paf", "tests")
-        filterPolicy = pexPolicy.Policy.createPolicy(
-            filterPolicyFile, filterPolicyFile.getRepositoryPath(), True)
-        imageUtils.defineFiltersFromPolicy(filterPolicy, reset=True)
+        imageUtils.defineFilter("g", 468.6)
+        imageUtils.defineFilter("r", 616.5)
 
+        unkFilter = afwImage.Filter()
+        gFilter = afwImage.Filter("g")
+        rFilter = afwImage.Filter("r")
         unkFilter = afwImage.Filter()
         gFilter = afwImage.Filter("g")
         rFilter = afwImage.Filter("r")
```

Comment:
Did you stutter?

Reply:
Good catch, don't know how that happened!